crowd-server/crowd_server/apps/food_fact/serializers.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import serializers
+from .models import *
+
+
+class QuestionSerialiser(serializers.ModelSerializer):
```

Comment:
Rename QuestionSerialiser to QuestionSerializer

Reply:
Thanks for your comments